bin/oscapd-cli

```diff
@@ -332,13 +403,39 @@ def confirm(prompt):
     sys.stderr.write("Operation aborted.\n")
     return False
 
-
-
 def main():
     parser = argparse.ArgumentParser(
         description="OpenSCAP-Daemon command line interface."
     )
     subparsers = parser.add_subparsers()
+    task_accessor = Task_accessor()
```

Comment:
Check out PEP8 and class names, `Task_accessor` is not camelcase - https://www.python.org/dev/peps/pep-0008/#class-names

I find the design a bit strange, `Task_accessor` has some static methods and some regular methods, yet it is for all intents and purposes a singleton initialized only here.